sonar-plugin-api/src/main/java/org/sonar/api/issue/Issuable.java

```diff
@@ -112,8 +113,10 @@
     IssueBuilder severity(@Nullable String severity);
 
     /**
-     * Login of the user who reported the issue. Optional.
+     * @deprecated since 5.5, manual issue feature has been dropped.
      */
+    @Deprecated
+    @CheckForNull
     IssueBuilder reporter(@Nullable String reporter);
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)